src/stores/ChannelStore.js

```diff
@@ -96,6 +96,21 @@ class ChannelStore extends DataStore {
    * @param {ChannelResolvable} channel The channel resolvable to resolve
    * @returns {?Snowflake}
    */
+
+  /**
+    * Obtains a channel from Discord, or the channel cache if it's already available.
+    * <warn>This is only available when using a bot account.</warn>
+    * @param {Snowflake} id ID of the channel
+    * @param {boolean} [cache=true] Whether to cache the new channel object if it isn't already
+    * @returns {Promise<GuildChannel>}
```

Comment:
This would be a `Channel`, as this data store is used for more than guild channels.

Reply:
Fixing now.